index.html

```diff
@@ -1290,32 +1290,35 @@ <h2>DID Subject</h2>
 party going forward.
       </p>
 
-      <p>
+    <section>
+      <h3>alsoKnownAs</h3>
+
+        <p>
 A <a>DID subject</a> can have multiple identifiers for different purposes, or
 at different times. The assertion that two or more <a>DIDs</a> (or other types
```

Comment:
I know its not in this PR, but for me the word assertion implies cryptography... I see this property as a claim made by the did controller, not an assertion.